skafka/src/main/scala/com/evolutiongaming/skafka/OffsetAndMetadata.scala

```diff
@@ -19,7 +19,7 @@ object OffsetAndMetadata {
   implicit val showOffset: Show[OffsetAndMetadata] = Show.fromToString
 
   implicit val orderOffset: Order[OffsetAndMetadata] =
-    Order.whenEqual(Order.by { a: OffsetAndMetadata => a.offset }, Order.by { a: OffsetAndMetadata => a.metadata })
+    Order.whenEqual(Order.by(_.offset), Order.by(_.metadata))
```

Comment:
is this also a problem for Scala 3?

Reply:
The problem was lack of `()` around arguments in lambdas in Scala 3. Instead of rewriting it to `(a: OffsetAndMetadata) => a.offset` I decided to make it shorter.

Reply:
thanks